<launch>
    <!-- <include file="$(find hephaestus_moveit)/launch/hephaestus_moveit.launch"/> -->
    <!-- <node name="robot_state_publisher" type="robot_state_publisher" pkg="robot_state_publisher"/> -->
    
    <include file="$(find realsense2_camera)/launch/rs_camera.launch"/>

    <node pkg="aruco_ros" type="single" name="aruco_single">
        <remap from="/camera_info" to="/camera/color/camera_info" />
        <remap from="/image" to="/camera/color/image_raw" />
        <param name="image_is_rectified" value="True"/>
        <param name="marker_size"        value="0.1"/> <!-- meter -->
        <param name="marker_id"          value="582"/>
        <param name="reference_frame"    value="camera_color_optical_frame"/>   <!-- frame in which the marker pose will be refered -->
        <param name="camera_frame"       value="camera_color_optical_frame"/>
        <param name="marker_frame"       value="marker_link" />
        <param name="corner_refinement"  value="LINS" /> <!-- NONE, HARRIS, LINES, SUBPIX -->
    </node>

    <!-- <include file="$(find easy_handeye)/launch/calibrate.launch" >
        <arg name="namespace_prefix" value="" />
        <arg name="move_group" value="hephaestus_arm_group" />
        <arg name="eye_on_hand" value="false" />
        <arg name="tracking_base_frame" value="camera_color_optical_frame" />
        <arg name="tracking_marker_frame" value="marker_link" /> 
        <arg name="robot_base_frame" value="link1" />
        <arg name="robot_effector_frame" value="gripper" />

        <arg name="freehand_robot_movement" value="false" />
        <arg name="robot_velocity_scaling" value="0.5" />
        <arg name="robot_acceleration_scaling" value="0.2" />
    </include> -->

</launch>